overlord/snapshotstate/backend/backend.go

```diff
@@ -589,7 +593,43 @@ func writeOneSnapshotFile(targetPath string, tr io.Reader) error {
 	return nil
 }
 
-func unpackVerifySnapshotImport(r io.Reader, realSetID uint64) (snapNames []string, err error) {
+type DuplicatedSnapshotImportError struct {
```

Comment:
Does it have to be exported?

Reply:
A good question, I was thinking it might be useful in the clients but maybe YAGNI and all that?